dietpi/dietpi-software

```diff
@@ -13853,7 +14009,15 @@ _EOF_
 			fi
 			# - Nginx
 			[[ -d '/etc/nginx/sites-dietpi' ]] && rm -f /etc/nginx/sites-dietpi/dietpi-pihole*
+      
+			# Unbound
+			
+			if (( ${aSOFTWARE_INSTALL_STATE[182]} > 1 )); then
```

Comment:
Check first if the file actually exists. It should when not touched manually, but just to be sure that no obsolete info + error message is shown here.